caluma/form/validators.py

```diff
@@ -81,7 +81,7 @@ def validate(self, *, form, answers, **kwargs):
         answer_by_question = {answer.question.slug: answer.value for answer in answers}
```

Comment:
The way this is written, you only get access to the answers currently being validated. Is this what you want? Or wouldn't it be better to have all questions of the form available (and only look them up from the QS when needed)?

Obviously this would require some overloading, so you get the "new" answer if available, etc..

Reply:
all answers always get validated when whole document gets validated (reason it could be that configuration of a question has changed after a answer has already given but before document is being completed). Hence what you are describing is already the case.